CONTRIBUTING.md

```diff
@@ -168,6 +168,10 @@ tox -e py38 -- test/cli
 tox -e py38 -- test/cli/commands_test.py
 ```
 
+#### dbt templater tests
+
+The dbt templater tests require a locally running Postgres instance. See the required connection parameters in `plugins/sqlfluff-templater-dbt/test/fixtures/dbt/profiles.yml`. We recommend using https://postgresapp.com/.
```

Comment:
+1 for providing a Docker Compose configuration for running the dbt tests. I actually have some bits of this on my local machine already, because prior to getting dbt 1.0 working, I couldn't install dbt (older versions) at all.

If we want to create a separate issue for setting up Docker Compose, I can potentially take on part of that work, or at least share what I have done so far.

Reply:
Amazing, could you create the issue and share what you've got already? Can just take the motivation/description from my comment above 😄

Reply:
Done! #2248

Reply:
Done!

Reply:
thanks :)

Reply:
@barrywhart actually looks like you haven't pushed the doc changes?

Reply:
Oh yes, I pushed them just now. I usually push to my fork, but with this PR, I need to push to the main SQLFluff repo.